src/styles/blocks/_review.scss

```diff
@@ -0,0 +1,40 @@
+.review {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  align-content: flex-end;
+
+  width: $card-width-without-border * 0.83;
+  margin-bottom: $card-width-without-border * 0.12;
```

Comment:
same here with variables as below, don't do that
also you can't add external styles to block, 8 point in checklist